stock_picking_backorder_strategy/README.rst

```diff
@@ -0,0 +1,70 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Picking backordering strategies
+===============================
+
+In types of operation add a backorder strategy to be able to cancel created
+backorder or to avoid to create a backorder.
+
+Possible strategies are :
```

Comment:
Nitpicking of nitpicking: `:` doesn't have a space before it in english. It's only in french that it needs to be there.

Same for the 3 next lines